scripts/mkjson.mjs

```diff
@@ -0,0 +1,15 @@
+import { writeFile } from 'fs/promises'
+import { join } from 'path';
+//A quick script to regenerate package.jsons for each cjs and esm after tsup cleans distributions
```

Comment:
For future, please add a space between // and comment, it just looks nicer.

Reply:
👍